tests/test_search.py

```diff
@@ -165,10 +165,15 @@ def test_doc_by_field(self):
         self.assertEqual(self.searcher.doc('CACM-3134').docid(),
                          self.searcher.doc_by_field('id', 'CACM-3134').docid())
 
+    def test_phrase_df(self):
+        self.assertEqual(74, self.searcher.phrase_df("inform retriev"))
+        self.assertEqual(135, self.searcher.phrase_df("search"))
+        
```

Comment:
can you add assert that confirms phrase df for a single term is the same as the original method?

Actually, now that I think about it... why do we need both methods? What about one method that either takes phrases or individual terms?

Reply:
What are you exactly referring to when you say 'original method'? If you mean IndexReaderUtils' `get_term_counts`, then it would be more intuitive to have the phrase_df function in the IndexReaderUtils I think.

Reply:
yea, do you think it'd make more sense to but directly in `get_term_counts` and allow it to take phrases as well?

Reply:
want me to refactor it in Anserini and move it to that method?

Reply:
Yes! That is exactly how I imagined to use it.

Reply:
Let's do it then?

Reply:
but we'll also need to get cf from a phrase?

Reply:
I can get all hits from the phrase query, loop over the documents and sum the tf for each hit?

Reply:
Actually I am not sure if you can get the tf easily given a phrase and a document..

Reply:
If it's not easy, then make sure we return `None`?

Reply:
Ok!